src/Vision/ImageAnnotations.php

```diff
@@ -69,12 +69,12 @@ public function setText(?string $text): void
 
     public function isIdentityDocument(): bool
     {
-        return 0 < array_intersect($this->labels, [
+        return 0 < \count(array_intersect($this->webEntities, [
```

Comment:
is it normal that the method `isIdentityDocument` uses `$this->webEntities` and `isFrenchPassport` uses `$this->labels`

Reply:
yes it is ! 
i look at both of these arrays to make sure its a document